### ISSUE TITLE
feat: after creating a hypercert the user should be dropped on their hypercert's page

### ISSUE THREAD

**Issue by ccerv1** [hypercerts-org/hypercerts#406]
### Describe the feature you'd like to request

Once the hypercert minting is finished, the user should be taken to their newly created hypercert page (instead of the project dashboard which most likely will be blank and gives no instruction of what to do next).

Here is a Loom of the [current UX](https://www.loom.com/share/7d750ecc515e4e3e933718d7186b8fe2)

### Describe the solution you'd like

Once the minting is complete...

- It would be nice to have some confetti or other fun way of celebrating (see #258)
- The creator should be dropped on their hypercert's page, not the dashboard
- The creator should have an option from that page to share on social (see #405)

### Describe alternatives you've considered

Keep the dashboard, but add a label for projects you've created under "Your Stats"

<img width="1182" alt="image" src="https://user-images.githubusercontent.com/42869436/221553466-674d8d40-f445-41f1-9bfa-1780d692d084.png">

**Comment by ccerv1** [comment]
Deprioritized. The user will now be dropped on their dashboard, with their hypercerts sorted by age so the newly created cert appears first on the list
